corehq/apps/userreports/templates/userreports/data_source_from_app.html

```diff
@@ -5,12 +5,12 @@
 
 {% block js-inline %}{{ block.super }}
     <script type="text/javascript">
-        $(function(){
-            ko.applyBindings({
-                optionsMap: ko.observable({{ data_source_options_map|JSON }}),
-                appId: ko.observable()
-            }, $('#data-source-config').get(0));
-        });
+        var viewModel = {
```

Comment:
nope, just copied it from the other file and didn't think about it. https://github.com/dimagi/commcare-hq/commit/beacc5234221a2c75712013313f2793240588666